### PR TITLE
Bump CloudBees Folder from 6.3 to 6.15

### PR DESCRIPTION
**Pull request by basil**
Bumps [CloudBees Folder](https://github.com/jenkinsci/cloudbees-folder-plugin) from 6.3 to 6.15. See [the release notes](https://github.com/jenkinsci/cloudbees-folder-plugin/releases).

### CODE REVIEW

**Review by res0nance — APPROVED** [review]

**Review by timja — APPROVED** [review]

**Comment by timja** [comment]
This PR is now ready for merge, after ~24 hours, we will merge it if there's no negative feedback.

Thanks!

**Review by StefanSpieker — APPROVED** [review]

**Pull request merged**
